overlays/unnecessary-parallel-building.nix

```
@@ -9,8 +9,12 @@ let
       cond =
         let
           inputs = drvArgs.nativeBuildInputs or [ ] ++ drvArgs.propagatedBuildInputs or [ ] ++ drvArgs.nativeBuildInputs or [ ];
+          toolInInputs = (lib.elem prev.meson inputs || lib.elem prev.cmake inputs || lib.elem prev.qt5.qmake inputs);
```

Maybe something like

```suggestion
          toolConfigureHookNotDisabled = (lib.elem prev.meson inputs && !drvArgs.dontUseMesonConfigure or false) || (lib.elem prev.cmake inputs && !drvArgs.dontUseCmakeConfigure or false) || (lib.elem prev.qt5.qmake inputs && !drvArgs.dontUseQmakeConfigure or false);
```

so that stray `dontUse*Configure` flags do not confuse it.